casper/contracts/simple_casper.v.py

```diff
@@ -448,7 +445,15 @@ def highest_finalized_epoch(min_total_deposits: wei_value) -> int128:
 
 @public
 @constant
-def havent_voted_and_source_target_right(
+def in_dynasty(validator_index:int128, _dynasty:int128) -> bool:
+    start_dynasty: int128 = self.validators[validator_index].start_dynasty
+    end_dynasty: int128 = self.validators[validator_index].end_dynasty
+    return (start_dynasty <= _dynasty) and (_dynasty < end_dynasty)
+
+
+@public
+@constant
+def _votable(
```

Comment:
This should be @private. the `_` prefix indicates it as such, but we should hide it from the abi because we gave them the just as powerful and easier to use `votable`